app/models/chargeback_vm.rb

```diff
@@ -82,11 +82,7 @@ def self.build_results_for_report_ChargebackVm(options)
 
   def self.where_clause(records, options, region)
     scope = records.where(:resource_type => "VmOrTemplate")
-    if options[:tag] && (@report_user.nil? || !@report_user.self_service?)
-      scope.for_tag_names(options[:tag].split("/")[2..-1])
-    else
-      scope.where(:resource => vms(region))
-    end
+    scope.where(:resource => vms(region))
```

Comment:
I don't understand this change.  Why can we remove this?

Reply:
it was bad touch, now I extended it to posibility filter as according to current tags and also according to old tags.

this change is needed for filtering out  resource on report - related to filter tab in report